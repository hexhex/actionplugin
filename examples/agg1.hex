ok(C) :- #count{X,Y : a(X,Z,k),b(e,Z,Y)} = 3, p(C).
p(W) :- #min{S : c(S)} = W.
b(e,h,r) :- on.
on v off.
a(f,g,k).
a(u,h,k).
b(e,g,s).
b(e,h,x).
b(e,h,q).
c(2).
c(3).
c(6).
