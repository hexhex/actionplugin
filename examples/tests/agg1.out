{off, c(2), c(3), c(6), p(2), b(e,g,s), b(e,h,q), b(e,h,x), a(u,h,k), a(f,g,k), ok(2)}
{on, c(2), c(3), c(6), p(2), b(e,g,s), b(e,h,q), b(e,h,x), b(e,h,r), a(u,h,k), a(f,g,k)}
